statchatexpress/.gitignore

```diff
@@ -1,3 +1,2 @@
 node_modules
-package-lock.json
-package.json
\ No newline at end of file
+info.env
```

Comment:
Normally the file is called `.env`. These are called "dotfiles", which are conventionally used as configuration files.

Are things still working as you expect if you use `info.env`?

Reply:
yes, i added the username and password to the .env filen and then did git ignore on it so it wouldnt show. everything is working just fine with the .env file.